frontend/src/vault/send/verify/SendVerify.tsx

```diff
@@ -26,16 +33,22 @@ import { SendConfirm } from './SendConfirm';
 import { SendTerms } from './SendTerms';
 import { sendTermsCount, SendTermsProvider } from './state/sendTerms';
 
-export const SendVerify: React.FC<ComponentWithBackActionProps> = ({
-  onBack,
-}) => {
+export const SendVerify: FC<ComponentWithBackActionProps> = ({ onBack }) => {
   const { t } = useTranslation();
   const [coinKey] = useCurrentSendCoin();
   const sender = useSender();
   const coin = useCurrentVaultCoin(coinKey);
   const [receiver] = useSendReceiver();
   const [amount] = useSendAmount();
   const [memo] = useSendMemo();
+  const formatFiat = useFormatFiatAmount();
+  const { data: coinPrices = [] } = useCoinPricesQuery([
+    CoinMeta.fromCoin(storageCoinToCoin(coin)),
+  ]);
+  const price =
+    shouldBePresent(coinPrices).find(item => areEqualCoins(item, coinKey))
```

Comment:
You set it to [] by default, so it will be present anyway, no need for `shouldBePresent`